client/src/app/vets/components/vet-animals-type/vet-animals-type.component.scss

```diff
@@ -0,0 +1,46 @@
+@media(max-width: 770px){
+  
+  .animals-types__cards-set .item{
```

Comment:
Fix code formatting (spaces, tabulations) in this file

Reply:
Fixed with Prettier